### DISCUSSION TITLE
CSS larger in production environment

### DISCUSSION THREAD

**Discussion by taylor-lindores-reeves** [vercel/next.js#12787] [Help]
I am running `next build` then `next start` locally to replicate the production environment, and when I do so locally, here is what I see:

<img width="1680" alt="Screenshot 2020-05-12 14 15 30" src="https://user-images.githubusercontent.com/27443324/81695853-185e7600-945b-11ea-863d-1a311d1b980e.png">

However, when I do the exact same in my Node.js Quickstart server (production environment), you can see here https://www.digitalspeed.co.uk that everything is slightly different - logo is bigger, margins are smaller, font-size is bigger...

Any idea why this could be happening?

Thanks!

** UPDATE **

embarrassingly I had the localhost set to 90% which persisted throughout development, and then also when I have my laptop plugged into my desktop monitor it scales. Both of these reasons caused the magnification, so nothing to do with Next.js. Apologies!

**Comment by David-Melo** [comment]
I've had a similar issue and it ended up being the CSS load ordering for me. Sometimes in the final CSS bundle, the order of your CSS might not stay consistent and thus certain CSS properties will get changed depending on where the selectors appear in the final bundle.

**Reply by taylor-lindores-reeves** [reply]
Thanks, my question is why would it bundle differently depending on the environment? As in, I'm running the exact same commands just one is on my local env where the other is on my prod server.

**Reply by David-Melo** [reply]
That I'm not sure exactly. In production you have minification and other optimizations that might be applied to the css that you wouldn't get in dev. You could try to set the production flag in your dev machine to see if the behavior is consistent. 

I did find at least one other issue that might be the same behaviour.

https://github.com/zeit/next.js/issues/10148

**Comment by Timer** [comment]
What Next.js version are you on?

**Reply by taylor-lindores-reeves** [reply]
9.3.6-canary.0

**Reply by Timer** [reply]
Can you please try the latest Next.js version? We fixed some bugs regarding CSS ordering in dev.

**Reply by taylor-lindores-reeves** [reply]
Sure thing, I have just updated to 9.4.0 but its still the same. Everything is magnified. I've triple checked the CSS and the strange thing is that it's exactly the same, but in production it's larger. For example, the width of the logo is 300px, but in the production environment it is magnified.

Oh, and one other thing I just noticed, is that when I inspect and toggle the Chrome device toolbar, both local and prod look identical. However, when I turn off the console on both, the magnification persists.

**Reply by tomthornton** [reply]
Was a solution ever found for this? My entire app is larger in production than it is in development
